modules/integration_gcp-load-balancing/variables-gen.tf

```diff
@@ -212,7 +212,7 @@ variable "backend_latency_service_notifications" {
 variable "backend_latency_service_aggregation_function" {
   description = "Aggregation function and group by for backend_latency_service detector (i.e. \".mean(by=['host'])\")"
   type        = string
-  default     = ".sum(by=['forwarding_rule_name', 'backend_target_name'])"
+  default     = ".mean(by=['forwarding_rule_name', 'backend_target_name'])"
```

Comment:
To also do in the configuration file used by templating: https://github.com/claranet/terraform-signalfx-detectors/blob/1c2cecad3d0820044086a245ea84323f5a35da0f/modules/integration_gcp-load-balancing/conf/03-latency-service.yaml#L6